### PR TITLE
[phone-call] JSRS migration vcap 24499

### PR DESCRIPTION
**Pull request by gabrieloliveira41**

### CODE REVIEW

**Comment by gabrieloliveira41** [comment]
the migration tool successfully migrated the code

**Comment by gabrieloliveira41** [comment]
Goals verified after changes (2 out of 2 goals total):
- FindBusiness
start
Find businesses
- CallingInfo#call
Call this business

All resulted in equal intents and equal IDE simulator results after migration comparing to before the migration

**Comment by gabrieloliveira41** [comment]
there is only one warning left 
image
but i looked on the dev center and the `bixby.phonecall` recomended version is exactly this one `1.0.12`
image

**Comment by gabrieloliveira41** [comment]
Hello @kangawoo44 @abapayeViv, could you review my PR, please?

**Comment by gabrieloliveira41** [comment]
just an info when you press the `call` button 
image
the capsule asks for a mock data and that is happening on both versions after and before the migration 
image

**Comment by gabrieloliveira41** [comment]
i checked on our side and there's not much we can do cause we do not have access to bixby.phoneCall , please let me know if there's anything else i can do from here

File: library-phone-call-example/capsule.bxb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 capsule {
   id (example.phoneCallExample)
-  version (0.1.2)
+  version (0.2.0)   //JSRS migration VCAP-24499
   format (3)
   targets {
     target (bixby-mobile-en-US)
   }
   runtime-version (7) {
```
Comment (abapayeViv) [inline library-phone-call-example/capsule.bxb:8]: Can we update this to runtime-version (9)?

**Comment by gabrieloliveira41** [comment]
image
the same warning message is shown in the capsule before the migration

**Review by kangawoo44 — APPROVED** [review]
since these warnings were there before the JSRS migration we'll merge this change and work on fixing the library warnings separately.

**Comment by gabrieloliveira41** [comment]
Hello @abapayeViv, could you review my PR, please?

**Comment by gabrieloliveira41** [comment]
Hello @abapayeViv, could you review my PR, please?

**Review by abapayeViv — APPROVED** [review]
👍

**Comment by gabrieloliveira41** [comment]
@abapayeViv, could you merge this PR?

**Pull request merged**